### DISCUSSION TITLE
Question about recommended CUDA and Pytorch version & LAMMPS

### DISCUSSION THREAD

**Discussion by turbosonics** [ACEsuit/mace#327] [Q&A]
Hello! 

I'm about to install from my local cluster using venv. I see the recommendation of PyTorch >= 1.12 (training with float64 is not supported with PyTorch 2.1). But there are so many versions of pytorch for various Cuda. Any optimal environment recommendation for training/computation speed and memory allocation? Our local cluster supports Cuda 11.2 and 11.8. 

Also, I checked [this](https://github.com/ACEsuit/mace/blob/docs/docs/guide/lammps.rst) document about mace for LAMMPS, and I saw the big warning sign of "work in progress". Are there any specific parts, such as NPT or NVE ensemble, do I need to extra careful about LAMMPS Mace simulations? 

Thanks!

**Comment by wcwitt** [comment]
Hi, sorry for the slow reply. I believe CUDA 11.8 should work, although @ilyes319 would know best.

With respect to LAMMPS, you should not have any problems with basic usage, but don't hesitate to reach out if anything seems strange.
